pkg/winsvc/service_windows.go

```diff
@@ -0,0 +1,37 @@
+//go:build windows
+// +build windows
+
+package winsvc
+
+import (
+	wsvc "golang.org/x/sys/windows/svc"
+)
+
+type serviceWindows struct{}
+
+func init() {
+	isService, err := wsvc.IsWindowsService()
+	if err != nil {
+		panic(err)
```

Comment:
If I am not mistaken, if there is an error everything will crash, right? Maybe we should ignore the error, wdyt?